[LiveComponent][TwigComponent] Add support for dynamic template resolution (#3361)
symfony/ux: src/LiveComponent/CHANGELOG.md

    }
    ```

> - Add support for dynamic template resolution via the `FromMethod` attribute.

Reviewer: ```suggestion
- Add support for dynamic template resolution with `AsLiveComponent(template: FromMethod('getTemplate'))`
```

Author: - Add support for dynamic template resolution with `AsLiveComponent(template: FromMethod('customFunction'))`
 
 Does it make sense?

Reviewer: yes

> - Add support for dynamic template resolution with AsLiveComponent(template: FromMethod('customFunction')

Reviewer: ```suggestion
- Add support for dynamic template resolution with `AsLiveComponent(template: FromMethod('customFunction'))`
```
